subscriptions/follow_up/automatic_alerts.rst

```diff
@@ -0,0 +1,89 @@
+their dissatisfaction.
+
+With **Odoo Subscriptions**, you can set automatic emails, you can also create a "Call" task for one
+of your salespeople so that he/she can try to understand your customer's dissatisfaction, and finally,
+why not automatically send satisfaction surveys so customers can evaluate your services? All of that
+is now possible.
+
+Create a new automatic alert
+============================
+
+The following example shows how to create a new automatic alert to send satisfaction surveys to your
+customers, by email, after one month of subscription. To do so, go to
+:menuselection:`Subscriptions --> Configuration --> Alerts`, and create a new
+alert.
+
+.. image:: media/create-a-new-automatic-alert.png
+  :align: center
+  :alt: New automatic alert in Odoo Subscriptions
+
+1. On the *Apply on* section, first give the alert a name. Then, you can choose to apply this alert
+   on a subscription template, on a specific customer, or even on a specific product. If you want to
+   add more specifications, you can also specify the value of your MRR, the change rate of your MRR
+   over a certain period of time, the value of the satisfaction rate, and even the stage to which
+   you want to apply this alert.
+
+   .. note::
+      In this example, the alert is applied to a specific product, and the stage goes from *Undefined*
+      to *In progress*.
+
+2. For the *Action* section, specify the *Action* and the *Trigger on*. If the *Trigger on* is set to
+   *Modification*, the action is triggered every time there is a change or anything added to the
+   subscription, and all the conditions on the *Apply on* section are met. Now, if the *Trigger on*
+   is set to *Timed condition*, it means that the action is triggered based on the type of
+   *Trigger date*. After that, you can choose your *Action*. You have the choice between
+   *Create next activity*, *Set a tag on the subscription*, *Set a stage on the subscription*,
+   *Mark as To Renew*, *Send an email to the customer* and *Send an SMS Text Message to the customer*.
+
+   .. note::
+      In the example above, the *Trigger on* is set to *Timed condition*, therefore, a *Trigger date*
+      and *Delay after trigger* need to be specified. And because the *Send an email to the customer*
+      action was adopted, an *Email template* can be chosen.
+
+As a result, this alert will send a rating survey after one month, to the customers who have purchased
+that specific product. The survey will appear in the chatter of your respective subscription.
+
+.. image:: media/rating-satisfaction-survey.png
+  :align: center
+  :alt: Satisfaction survey in Odoo Subscriptions
+
+
+Modify an existing automatic alert
+==================================
+
+By default, Odoo suggests you an automatic alert called *Take action on less satisfied clients*.
+
+.. image:: media/modify-an-existing-automatic-alert.png
```

Comment:
I would try to get rid of a bit more of the white spaces, in order to make the image be a bit more easily 'readable'.

Reply:
I already used GIMP to get rid of these white spaces... But the purple header is taking too much space, even if I restore down my window... :/